ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/intended/data/test-ids.yml

```diff
@@ -0,0 +1,113 @@
+# This file is auto-generated by AVD eos_designs.
+# When there is a merge conflict, rebase the branch, accept all current changes, and re-run AVD.
+id_pools:
```

Comment:
Would it be possible to indent the list with two spaces so that it allows to collapse each list easily (in IDE)?

Reply:
Nope. We use PyYAML with integrated libyaml for performance. The only way to render the extra spaces is to use a custom dumper in pyyaml (loosing libyaml performance), or change to ruamel.yaml which would be a new requirement just for this.